eth2deposit/credentials.py

```diff
@@ -91,10 +98,11 @@ def export_deposit_data_json(*, credentials: List[ValidatorCredentials], folder:
             withdrawal_credentials=credential.withdrawal_credentials,
             amount=credential.amount,
         )
-        signed_deposit_datum = sign_deposit_data(deposit_datum, credential.signing_sk)
+        signed_deposit_datum = sign_deposit_data(deposit_datum, credential.signing_sk, credential.fork_version)
         datum_dict = signed_deposit_datum.as_dict()
         datum_dict.update({'deposit_data_root': deposit_datum.hash_tree_root})
         datum_dict.update({'signed_deposit_data_root': signed_deposit_datum.hash_tree_root})
+        datum_dict.update({'fork_version': credential.fork_version})
```

Comment:
Note that to verify it wtih `verify_deposit_data_json`, this PR adds `fork_version` in the `deposit_data*.json` file.

/cc @CarlBeek

Reply:
For future reference (as @hwwhww has already approved the change that fixes this):

It does break things, but it is neccicary to do so. ethereum/eth2.0-deposit/pull/84 resolves this